sapi/fpm/php-fpm.conf.in

```diff
@@ -41,6 +41,24 @@
 ; Default Value: notice
 ;log_level = notice
 
+; Log limit on number of characters in the single line (log entry). If the
+; line is over the limit, it is wrapped on multiple lines. The limit is for
+; all logged characters including message prefix and suffix if present. However
+; the new line character does not count into it as it is present only when
+; logging to a file descriptor. It means the new line character is not present
+; when logging to syslog.
+; Default Value: 1024
+;log_level = 4096
```

Comment:
Shouldn't this say `log_limit` instead of `log_level`?

Reply:
Ah good catch! Will fix it! Thanks!

Reply:
Should be fixed now!